bundle_result.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package bundle_result;
+
+// Indicates the bundle was forwarded to validators.
+message Forwarded {
+  // Maps slots to validator pubkeys this bundle was forwarded to.
+  map<uint64, string> validator_identities = 1;
+}
+
+// Indicates the bundle was dropped and therefore not forwarded to any validator.
+message Dropped {
+  oneof reason {
+    AuctionFailure auction_failure = 1;
+    SimulationFailure simulation_failure = 2;
+    InternalError internal_error = 3;
+  }
+}
+
+
+message AuctionFailure {
+  // Auction's unique identifier.
+  string auction_id = 1;
+
+  // Bundle's simulated bid.
+  uint64 simulated_bid_lamports = 2;
+}
+
+// Bundle dropped due to simulation failure.
+message SimulationFailure {
+  // Signature of the offending transaction.
+  string tx_signature = 1;
+  optional string msg = 2;
```

Comment:
what is msg here? 

logs from execution? looking at struct:
```
pub struct RpcSimulateTransactionResult {
    pub err: Option<TransactionError>,
    pub logs: Option<Vec<String>>,
    pub accounts: Option<Vec<Option<UiAccount>>>,
    pub units_consumed: Option<u64>,
    pub return_data: Option<UiTransactionReturnData>,
}
```

or 
```
pub struct RpcSimulateBundleResult {
    pub summary: RpcBundleSimulationSummary,
    pub transaction_results: Vec<RpcSimulateBundleTransactionResult>,
}
```

for errors, could piggyback on https://github.com/jito-foundation/jito-solana/blob/master/storage-proto/proto/transaction_by_addr.proto#L21 but might unnecessarily complicate build system

Reply:
yea kind wanna leave it to be just some arbitrary err msg searchers can manually tail